src/main/java/module/TicTacToeService.java

```diff
@@ -0,0 +1,157 @@
+package main.java.module;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class TicTacToeService {
+    TicTacToeBoard ticTacToeBoard = new TicTacToeBoard();
+    Queue<Player> playerSequence;
+
+    void setPlayerSequence(List<Player> players){
+        this.playerSequence = new LinkedList<>();
+        for(Player player : players){
+            playerSequence.add(player);
+        }
+    }
+    void initializeBoard(int row,int col){
+        ticTacToeBoard.setRow(row);
+        ticTacToeBoard.setCol(col);
+        ticTacToeBoard.initboard(row, col);
+
+    }
+    void setBoardValue(Player player,int row, int col){
+        ticTacToeBoard.setBoard(row-1, col-1, player.getSymbol());
+    }
+    void printBoardValues(){
+        for(int i = 0; i < ticTacToeBoard.getRow(); i ++){
+            for(int j = 0; j < ticTacToeBoard.getCol(); j++){
+                System.out.print(ticTacToeBoard.getBoard(i, j) + " ");
+            }
+            System.out.println();
+        }
+    }
+    boolean colWiseCheck(){
```

Comment:
Instead of count, you can have a function that given a row and col value checks if a given player has won or not. That method will check for col, row and both diagonals. You don't need counts. If any of the cell does not have that player then you can return false.